sycl/include/sycl/builtins.hpp

```diff
@@ -122,10 +122,27 @@ __SYCL_MATH_FUNCTION_OVERLOAD_FM(log2)
 __SYCL_MATH_FUNCTION_OVERLOAD_FM(log10)
 __SYCL_MATH_FUNCTION_OVERLOAD_FM(sqrt)
 __SYCL_MATH_FUNCTION_OVERLOAD_FM(rsqrt)
+__SYCL_MATH_FUNCTION_OVERLOAD_FM(fabs)
```

Comment:
I suggest to outline such tiny additions into a separate PR, because we can merge them almost right away.